### ISSUE TITLE
Breaking Authentication

### ISSUE THREAD

**Issue by riipandi** [stefanbauer/tailwindcss-breakpoint-detector#1]
Hi Sefan, 

I love Laravel and TailwindCSS, thank you for creating this package. I got a problem, when this package installed it's breaking my authentication process. When i submit the form, then redirected back to login form without any message.

FYI: I'm using login with username or email method.

![code](https://user-images.githubusercontent.com/921834/92324634-c4ad1380-f06d-11ea-96bb-f9748e5d6109.png)

Thanks.

**Comment by stefanbauer** [comment]
Are you sure it's because of this package? This package just hooks into the HTML and adds a div. That's it.

**Comment by riipandi** [comment]
Yeah, pretty sure. I remove this package and then authentication back to normal. May these problem caused by `InjectDetector`, so it's breaking the Request?

**Comment by BulgarianHealer** [comment]
It's also breaking the unique validation.
Seems like the imposter is the duplicated request mentioned above.
